code/workspaces/infrastructure-api/src/controllers/projectsController.js

```diff
@@ -0,0 +1,96 @@
+import { check, matchedData } from 'express-validator';
+import { service } from 'common';
+import projectsRepository from '../dataaccess/projectsRepository';
+
+async function listProjects(request, response, next) {
+  try {
+    const projects = await projectsRepository.getAll() || [];
```

Comment:
Do you need the `[]` I think that `getAll` will return an empty array if none are found...not 100% sure though.

Reply:
Right you are. I could have sworn that it didn't when I first added the method but it is now. Maybe I did something wrong elsewhere first time round! I will remove this.